spot-ml/src/main/scala/org/apache/spot/lda/SpotLDAWrapper.scala

```diff
@@ -39,6 +39,21 @@ import scala.collection.immutable.Map
 
 object SpotLDAWrapper {
 
+  /**
+    * Runs Spark LDA and returns a new model.
+    *
+    * @param sparkSession       the SparkSession
+    * @param docWordCount       RDD with document list and the word count for each document (corpus)
+    * @param topicCount         number of topics to find
+    * @param logger             application logger
+    * @param ldaSeed            LDA seed
+    * @param ldaAlpha           LDA alpha parameter
+    * @param ldaBeta            LDA beta parameter
+    * @param ldaOptimizerOption LDA optimizer, em or online
+    * @param maxIterations      number of maximum iteration LDA will run
```

Comment:
ldaAlpha  - shouldn't this be explained as document concentration
ldaBeta - should this be explained as topic concentration?